### PR TITLE
Update libraries and its reference in the app.

### PR DESCRIPTION
**Pull request by nishrana**

### CODE REVIEW

File: app/assets/sass/design_system.scss [hunk 1]
```diff
@@ -60,9 +60,9 @@
 @import '../../modules/elements/assets/sass/data-widgets';
 @import '../../modules/elements/assets/sass/category-widget';
 
-@import '../../bower_components/slick-carousel/slick/slick';
```
Comment (nishrana) [inline app/assets/sass/design_system.scss]: Fixed the path. Steve and I were getting sass compilation error.

File: app/modules/graphs/templates/graphs-c3.html [hunk 2]
```diff
@@ -4,9 +4,9 @@
     <div class="col-md-8">
       <h1 id="c3-charts" class="doc-head">C3.js </h1>
 
-      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="blank">angular-chart v0.3.2</a>
-      <a class="more-info plugin" href="" target="blank">D3 v3.5.0</a>
-      <a class="more-info plugin" href="http://c3js.org/" target="blank">C3 v~0.4.x</a>
+      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="blank">angular-chart v0.8.8</a>
+      <a class="more-info plugin" href="" target="https://d3js.org/">D3 v3.5.0</a>
```
Comment (danielcollettepb) [inline app/modules/graphs/templates/graphs-c3.html]: the target should be `_blank` and the url in the href

File: bower.json [hunk 3]
```diff
@@ -4,16 +4,16 @@
   "dependencies": {
     "Chart.js": "1.0.2",
     "Numeric-Stepper": "*",
-    "angular": "1.4.5",
+    "angular": "1.4.9",
     "angular-animate": "1.4.5",
```
Comment (danielcollettepb) [inline bower.json]: this should be 1.4.9

File: bower.json [hunk 4]
```diff
@@ -4,16 +4,16 @@
...
     "angular-bootstrap": "0.14.3",
     "angular-bootstrap-toggle-switch": "0.5.2",
     "angular-chart": "0.3.2",
-    "angular-chart.js": "0.7.2",
+    "angular-chart.js": "0.8.8",
     "angular-cookies": "1.4.5",
```
Comment (danielcollettepb) [inline bower.json]: this should be 1.4.9

File: bower.json [hunk 5]
```diff
@@ -4,16 +4,16 @@
...
     "angular-datatables": "0.5.1",
-    "angular-daterangepicker": "0.2.1",
-    "angular-dynamic-locale": "0.1.28",
+    "angular-daterangepicker": "0.2.2",
+    "angular-dynamic-locale": "0.1.30",
     "angular-highlightjs": "0.5.1",
     "angular-i18n": "1.4.5",
```
Comment (danielcollettepb) [inline bower.json]: this should be 1.4.9

File: bower.json [hunk 6]
```diff
@@ -23,13 +23,13 @@
     "angular-resource": "1.4.5",
```
Comment (danielcollettepb) [inline bower.json]: this should be 1.4.9

File: app/modules/graphs/templates/graphs-c3.html [hunk 7]
```diff
@@ -4,9 +4,9 @@
     <div class="col-md-8">
       <h1 id="c3-charts" class="doc-head">C3.js </h1>
 
-      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="blank">angular-chart v0.3.2</a>
-      <a class="more-info plugin" href="" target="blank">D3 v3.5.0</a>
-      <a class="more-info plugin" href="http://c3js.org/" target="blank">C3 v~0.4.x</a>
+      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="blank">angular-chart v0.8.8</a>
+      <a class="more-info plugin" href="https://d3js.org/" target="blank">D3 v3.5.0</a>
```
Comment (nishrana) [inline app/modules/graphs/templates/graphs-c3.html]: Added URL for d3 website similar to c3

File: bower.json [hunk 8]
```diff
@@ -23,13 +23,13 @@
...
     "angular-sanitize": "1.4.5",
```
Comment (danielcollettepb) [inline bower.json]: this should be 1.4.9

File: app/modules/graphs/templates/graphs-c3.html [hunk 9]
```diff
@@ -4,9 +4,9 @@
     <div class="col-md-8">
       <h1 id="c3-charts" class="doc-head">C3.js </h1>
 
-      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="blank">angular-chart v0.3.2</a>
-      <a class="more-info plugin" href="" target="blank">D3 v3.5.0</a>
-      <a class="more-info plugin" href="http://c3js.org/" target="blank">C3 v~0.4.x</a>
```
Comment (nishrana) [inline app/modules/graphs/templates/graphs-c3.html]: Got rid of the`~`. I think it's not needed

Comment (danielcollettepb) [inline app/modules/graphs/templates/graphs-c3.html]: you're right... good catch

File: bower.json (hunk 3 above)
Comment (nishrana) [inline bower.json]: It is :-). Changed it from 1.4.5 to 1.4.9

**Comment by danielcollettepb** [comment]
I found a strange bug. Go to the Web > Tables > Paging and click on the demo button. In the Paging Example page, click on page 2 at the bottom of the screen. When you click on the second page it opens the xeditable website. This does not happen on the live DS site

File: app/modules/graphs/templates/graphs-c3.html [hunk 10]
```diff
@@ -4,9 +4,9 @@
...
+      <a class="more-info plugin" href="https://github.com/GraFiddle/angular-chart" target="_blank">angular-chart v0.8.8</a>
```
Comment (nishrana) [inline app/modules/graphs/templates/graphs-c3.html]: Changed the target to `_blank`

**Comment by nishrana** [comment]
try bower prune? Doesn't happen on my local.

**Comment by nishrana** [comment]
image

**Comment by nishrana** [comment]
I'm on page 2 with no console errors.

File: bower.json (hunk 3 above)
Comment (nishrana) [inline bower.json]: done :+1:

File: bower.json (hunk 4 above)
Comment (nishrana) [inline bower.json]: done :+1:

File: bower.json (hunk 5 above)
Comment (nishrana) [inline bower.json]: done :+1:

File: bower.json (hunk 6 above)
Comment (nishrana) [inline bower.json]: done :+1:

File: bower.json (hunk 8 above)
Comment (nishrana) [inline bower.json]: done :+1:

**Comment by danielcollettepb** [comment]
strange... still happens on mine. Will have to look at this tomorrow.

**Comment by nishrana** [comment]
hmm..ok

**Comment by lombardi-gux** [comment]
I had something similar happen once when using the jump-links.  For no apparent reason, the page would go to the UI-bootstrap web site.  I cleaned bower folder and it stopped.


________________________________

**Comment by nishrana** [comment]
That's really interesting @smlombardi

**Comment by danielcollettepb** [comment]
@smlombardi tried deleting/reinstalling and the issue persists

**Comment by danielcollettepb** [comment]
@nishrana showed that the bug I am seeing is not happening on her machine, must be something on my Mac causing the problem. Going to merge this in...

**Comment by danielcollettepb** [comment]
FYI - blew away the node_modules folder and reinstalled... the tab opening issue is resolved on my machine

**Pull request merged**